packages/testing/src/execution_testing/cli/pytest_commands/plugins/shared/benchmarking.py

```diff
@@ -48,6 +83,32 @@ def pytest_generate_tests(metafunc: pytest.Metafunc) -> None:
                 "gas_benchmark_value", gas_parameters, scope="function"
             )
 
+    if "fixed_opcode_count" in metafunc.fixturenames:
```

Comment:
My only concern here is that `--gas-benchmark-values` can be used with this new parameter, however I don't understand the interplay between these two. I.e. can you still guarantee that a certain amount of gas is used if you constrain the amount of times an opcode is executed? I feel these two are mutually exclusive, so IMO we should only parametrize by one of them.